src/otp-input/types.ts

```diff
@@ -0,0 +1,20 @@
+import { ButtonProps } from "src/button";
+
+export interface OtpInputProps extends React.AriaAttributes {
+    value?: string | undefined;
```

Comment:
add `id`, `className` props as well. `className` is important else the user can't modify the style